### PR TITLE
Add some usage documentation to scripts

### PR DESCRIPTION
**Pull request by RyanrDavies**
Adds some more usage documentation to scripts that were missing it, and a bit of a tidy up of the help messages so that they all have the same format.

### CODE REVIEW

File: free-gpus [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+Usage: $0
+    List nodes with free GPUs, as well as a breakdown of how many GPUs in each node are available.
+
+Arguments:
+    None.
+
+Output:
+    Lists each node with at least one free GPU on separate lines.
+    Also lists a breakdown of the states of the GPUs:
+        -- The 'in_use' column shows the number of GPUs on the node that are currently allocated to a job.
```
Comment (JamesOwers) [inline free-gpus]: You know... at some point we should change this command to actually return the number of free gpus there are available! Currently you have to do usable - in_use in your head (not a big ask, but doesn't make it entirely clear).

File: gpu-usage [hunk 2]
```diff
@@ -8,37 +8,47 @@ print_usage () {
...
+Arguments:
     -n    (optional) specify nodes, either comma separated list, or summarised.
           Mirrors slurm command -n usage in sinfo, and -w usage for squeue." 
     -h    (optional) do not print header
     -p    (optional) make output pretty, ignores -h if specified
 
+Output:
+    Outputs three columns summarising the current usage of GPUs on the cluster:
+        -- The 'in_use' column shows the number of GPUs that are currently allocated to a job.
+        -- The 'usable' column shows the number of GPUs that are not in an error state or otherwise unable to be
```
Comment (JamesOwers) [inline gpu-usage]: You know... at some point we should change this command to actually return the number of free gpus there are available! Currently you have to do usable - in_use in your head (not a big ask, but doesn't make it entirely clear).

Comment (RyanrDavies) [reply gpu-usage]: Yeah, I was thinking that when I was writing the documentation. I'll maybe look at it later today -- I guess it shouldn't be too complicated to do.

File: gpu-usage [hunk 3]
```diff
@@ -8,37 +8,47 @@ print_usage () {
 Usage: $0 [-n charles[01-19]]
```
Comment (JamesOwers) [inline gpu-usage]: We could change this to `-n nodelist`. I'm not gonna ask for that, I actually think this is clearer!

File: gpu-usage (hunk 2 above)
Comment (JamesOwers) [reply gpu-usage]: Def don't prioritise. If you do do, please may we retain the current columns, just so as it won't break my logging scripts?

**Pull request merged**